.github/workflows/update-branding-assets.yml

```diff
@@ -0,0 +1,211 @@
+name: Update Branding Assets
+
+# This workflow is triggered by the branding repository via repository dispatch
+# when new branding data is available. It receives a manifest and artifact ID,
+# downloads the assets, and updates the landing page accordingly.
+
+on:
+  repository_dispatch:
+    types: [branding-update]
+  workflow_dispatch:
+    inputs:
+      artifact-id:
+        description: 'GitHub artifact ID containing branding assets'
+        required: true
+        type: string
+      manifest:
+        description: 'JSON manifest describing the branding data'
+        required: true
+        type: string
+
+permissions:
+  contents: write
+  actions: read
+  pull-requests: write
+
+jobs:
+  update-branding:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+
+      - name: Parse manifest from dispatch
+        id: parse-manifest
+        uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea # v7.0.1
+        with:
+          script: |
+            let manifest, artifactId;
+            
+            if (context.eventName === 'repository_dispatch') {
+              manifest = context.payload.client_payload.manifest;
+              artifactId = context.payload.client_payload.artifact_id;
+            } else {
+              manifest = JSON.parse(process.env.MANIFEST_INPUT);
+              artifactId = process.env.ARTIFACT_ID_INPUT;
+            }
+            
+            core.setOutput('manifest', JSON.stringify(manifest));
+            core.setOutput('artifact_id', artifactId);
+            
+            core.info(`📦 Branding update received:`);
+            core.info(`  Version: ${manifest.version}`);
+            core.info(`  Commit: ${manifest.commit}`);
+            core.info(`  Artifact ID: ${artifactId}`);
+        env:
+          MANIFEST_INPUT: ${{ inputs.manifest }}
+          ARTIFACT_ID_INPUT: ${{ inputs.artifact-id }}
+
+      - name: Validate and parse manifest
+        id: validate-manifest
+        uses: ./.github/actions/validate-manifest
+        with:
+          manifest: ${{ steps.parse-manifest.outputs.manifest }}
+
+      - name: Download branding artifact
+        uses: actions/download-artifact@fa0a91b85d4f404e444e00e005971372dc801d16 # v4.1.8
+        with:
+          name: branding-assets-${{ steps.parse-manifest.outputs.artifact_id }}
+          path: /tmp/branding-assets
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          repository: hoverkraft-tech/branding
+          run-id: ${{ steps.parse-manifest.outputs.artifact_id }}
+
+      - name: Extract and organize assets
```

Comment:
this should be done in a dedicated action which we pass inputs comming from validate-manifest outputs